packages/shared/lib/app.ts

```diff
@@ -29,16 +30,17 @@ export const mnemonic = writable<Array<string>>(null)
 
 interface SendParams {
     amount: number
+    unit?: Unit
     address: string
     message: string
```

Comment:
I'm not sure we want to include this in the scope but I would love the idea we do.

Messages can be passed in the URL already although the Send.svetle component is not supporting it. It would make sense as part of this change we add it there as well. 

Unfortunately I can't allocate anymore time to this....but I thought it would be good to point this out.

Reply:
Does that mean that the message can be passed and gets included in the transaction even though it is not displayed on the UI? That wouldn't be a big issue as most useres don't really send data with firefly manually imo

Reply:
yeah I don't think that's happening, messages are just dropped atm...see Send.svetle:

`const triggerSend = (internal) => {
        closePopup()
        if (internal) {
            // We pass the original selectedSendType in case we are masquerading as
            // an internal transfer by a send to an address in one of our
            // other accounts. When the transfer completes it resets
            // the send params to where it was
            internalTransfer(from.id, to.id, amountRaw, selectedSendType === SEND_TYPE.INTERNAL)
        } else {
            send(from.id, address, amountRaw)
        }
    }
`

so to support it: 
- new ui component needs to be added 
- above function needs to be updated to pass message as parameter. Does send supports it?

I just didn't spent time figuring out how **send** function works. It seems to be external.

Reply:
I think including `message` is outside of the scope of this merge. It may even need some changes in the node bindings, or even wallet.rs itself?

Reply:
Wallet.rs as well as the binding already supports data attachments to messages in the `TransferOptions` param of the send function.
I hope this issue will transalte in a PR anytime soon: https://github.com/iotaledger/firefly/issues/1256